src/ray/common/pb_util.h

```diff
@@ -0,0 +1,32 @@
+#ifndef RAY_COMMON_PB_UTIL_H
+#define RAY_COMMON_PB_UTIL_H
```

Comment:
The `CreateJobTableData` is only called by `NodeManager`, should we put it to the `raylet` folder instead? 

If it's indeed a util function and will be called by other places, could we instead rename this file to `src/ray/util/gcs_util.h`? The name `pb_util.h` seems ambiguous.

Reply:
The `CreateJobTableData` is also used in GCS test.
 It's better to put this file to the `common` folder rather than GCS, because there may be more helper methods for protobuf in future.

Reply:
This `CreateJobTableData` function is specific to "job table of GCS", so I agree that it makes more sense to put it under `src/ray/gcs/util.h` and built into the `gcs` `cc_library`.
 
Regarding the common util functions for protobuf, we already have a `grpc_util.h`, which handles common data structures of protobuf. By "common", I mean not specific to any concrete `.proto` definitions. For those concrete proto definitions, better just put in their own modules.

Reply:
We already have `util/util.h`, it's kind of weird if we put it under src/ray/gcs/util.h`. Better to rename it  to `src/ray/gcs/pb_util.h`.